pkgs/by-name/av/avdump3/package.nix

```diff
@@ -0,0 +1,44 @@
+{ lib, stdenv, writeScriptBin, fetchzip, dotnet-runtime, zlib }:
+
+stdenv.mkDerivation rec {
+  pname = "avdump3";
+  version = "8293_stable";
+
+  src = fetchzip {
+    url = "https://cdn.anidb.net/client/avdump3/avdump3_8293_stable.zip";
+    hash = "sha256-H9Sn3I4S9CmymKIMHVagDy+7svHs285S3EJgYQo+ks0=";
+    stripRoot = false;
+  };
+
+  installPhase = ''
```

Comment:
```
runHook preInstall
# Your code
runHook postInstall
```

Reply:
Oops, forgot to commit this change, sorry! Pushed now.